cypress/integration/learn.js

```diff
@@ -31,7 +31,7 @@ describe('Learn Page', () => {
             cy.get('[data-test="card"]')
                 .first()
                 .should('contain', 'Working with MongoDB Transactions')
-                .click();
+                .click({ force: true });
```

Comment:
There were some Cypress issues where elements could become detached from the DOM based on a race condition. Here this card should continue to exist so it is safe to force. The re-render seems to be just hydrating in fonts from what I could dig up.